src/plots/cartesian/layout_attributes.js

```diff
@@ -280,16 +280,17 @@ module.exports = {
         pattern: {
             valType: 'enumerated',
             values: [DAY_OF_WEEK, HOUR, ''],
-            dflt: '',
             role: 'info',
             editType: 'calc',
             description: [
                 'Determines a pattern on the time line that generates breaks.',
-                'If *' + DAY_OF_WEEK + '* - Sunday-based weekday as a decimal number [0, 6].',
+                'If *' + DAY_OF_WEEK + '* - days of the week in English e.g. \'Sunday\' or `\sun\`',
```

Comment:
Let's add a note like "matching is case-insensitive and considers only the first three characters."

Reply:
Good call. Done in https://github.com/plotly/plotly.js/pull/4661/commits/05ab23ca6f8d51100f04c9bdc91e053d926f0c24.